main.nf

```diff
@@ -251,23 +251,25 @@ workflow {
         process_vcfs.out.idx.set { tool_indices }
         }
 
-    intersect(
-        tool_gzvcfs,
-        tool_indices,
-        script_dir_ch,
-        )
+    if (params.algorithm.size() > 1 || params.input_type == 'vcf') {
+        intersect(
+            tool_gzvcfs,
+            tool_indices,
+            script_dir_ch,
+            )
 
-    all_files = tool_gzvcfs.mix(tool_indices)
-        .flatten()
-        .collect()
+        all_files = tool_gzvcfs.mix(tool_indices)
+            .flatten()
+            .collect()
 
-    identified_gzvcfs = tool_gzvcfs.flatten()
-        .map{ [algorithm: getToolName(it), path: it] }
-        .collect()
+        identified_gzvcfs = tool_gzvcfs.flatten()
+            .map{ [algorithm: getToolName(it), path: it] }
+            .collect()
 
-    plot_vaf(
-        identified_gzvcfs,
-        all_files
-        )
+        plot_vaf(
+            identified_gzvcfs,
+            all_files
+            )
```

Comment:
**suggestion**: If I'm not mistaken, the VAF plotting module should still be able to run properly ever if only a single algorithm is selected; we may want to leave that outside of the conditional here

Reply:
OK that's fixed, but perhaps the output should go in it's own folder instead of `Intersect-BCFtools-...`?

### Updated NFTests: 
- `a_mini-somaticsniper`
    - plot_vaf output: `/hot/software/pipeline/pipeline-call-sSNV/Nextflow/development/unreleased/sfitz-fix-single-tool-runs/a_mini-somaticsniper/call-sSNV-8.1.0/S2-v1.1.5/Intersect-BCFtools-1.21/output/BPG-7.1.0_TWGSAMIN_S2-v1.1.5_adjVAF.png`
- `a_mini-all-tools-std-input`
    - log: `/hot/software/pipeline/pipeline-call-sSNV/Nextflow/development/8.1.0/sfitz-fix-single-tool-runs/log-nftest-20241115T232019Z.log`
- `a_mini-all-tools-vcf-input`
    - log: `/hot/software/pipeline/pipeline-call-sSNV/Nextflow/development/8.1.0/sfitz-fix-single-tool-runs/log-nftest-20241115T235205Z.log`